README.md

```diff
@@ -7,7 +7,7 @@ No code is the best way to write secure and reliable applications. Write nothing
 Start by not writing any code.
 
 ```
-
+ 
```

Comment:
Please use a character that corresponds to no character set. This one is using UTF-8